Research tool for handling changelog and versioning of components (#308)
CoherentLabs/GameUIComponents: .changeset/README.md

```diff
{
  "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
  "changelog": "@changesets/cli/changelog",
  "commit": false,
  "fixed": [],
  "linked": [],
- "access": "private",
+ "access": "public",
- "baseBranch": "main",
+ "baseBranch": "master",
  "updateInternalDependencies": "patch",
  "ignore": []
}
```

These changes are required so the tool compare the changes of some branch with master.

## How to bump version - [`npx changeset`](https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#add)

To bump packages versions we can use the `npx changeset` command. Running it will promp you couple of things.

1. Choose the packages you want to bump.
2. Choose which packages are having major, minor and patch update.
3. Type the changelog. **Note that the message you type in this step will be applied the same for the changelog of each package you are updating.** Because of that make sure you review the generated .md files before continue with the other steps.

The command will generate randomly named .md files for each package that you selected with the message you typed.

> In the language of the changeset too it will generate `changeset` which is the randomly generated .md file.

Author: ```suggestion
In the language of the changeset tool, generated `changeset` is a generated .md file with random name.
```
